app/src/js/utils/url-helper.js

```diff
@@ -66,6 +66,7 @@ export function getPersistentQueryParams(location = {}) {
  * @param {string} path the path to be passed to history.push
  */
 export function historyPushWithQueryParams(path) {
+  if (!history) return;
```

Comment:
Yeah this really only applies (or should only apply) when running in node i.e. running unit tests. This check probably isn't strictly necessary since I wasn't able to remove `browser-env`, but hopefully we'll get to that point soon, so might as well leave it in 🤷‍♀️